lightning/src/chain/channelmonitor.rs

```diff
@@ -2200,7 +2219,7 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 		false
 	}
 
-	fn would_broadcast_at_height<L: Deref>(&self, height: u32, logger: &L) -> bool where L::Target: Logger {
+	fn should_broadcast<L: Deref>(&self, logger: &L) -> bool where L::Target: Logger {
```

Comment:
nit: as you retouched the name, maybe it could be called `should_broadcast_holder_commitment`? imho marginally improving readability of this confusing part of the codebase where a wide array of transactions is constantly processed.